sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeHealthcareEntityUtilClient.java

```diff
@@ -584,4 +693,8 @@ private void throwIfCallingNotAvailableFeatureInOptions(AnalyzeHealthcareEntitie
                     serviceVersion, TextAnalyticsServiceVersion.V2022_05_01));
         }
     }
+
+    private Context enableSyncRestProxy(Context context) {
```

Comment:
Do we have a common util for this,s o that we don't repeat this for every *UtilClient?

Reply:
Yeah. I would move the the Util class.